inc/html.class.php

```diff
@@ -660,13 +653,11 @@ static function displayAjaxMessageAfterRedirect() {
    /**
     * Common Title Function
     *
-    * @param $ref_pic_link    Path to the image to display (default '')
-    * @param $ref_pic_text    Alt text of the icon (default '')
-    * @param $ref_title       Title to display (default '')
-    * @param $ref_btts        Extra items to display array(link=>text...) (default '')
-    *
-    * @return nothing
-   **/
+    * @param string $ref_pic_link   Path to the image to display (default '')
+    * @param string $ref_pic_text   Alt text of the icon (default '')
+    * @param string $ref_title      Title to display (default '')
+    * @param string $ref_btts       Extra items to display array(link=>text...) (default '')
+    */
```

Comment:
It returns void.

Reply:
echo getMassiveActionCheckBox returns a string and so it does getCheckbox which is called by getMassiveActionCheckBox, so at the end this function is returning a string

Reply:
same issue with echoes